test/base/SpecBase.scala

```diff
@@ -163,6 +164,13 @@ trait SpecBase
       List.empty
     )
 
+  val correctionRequest: CorrectionRequest =
+    CorrectionRequest(
+      Vrn("063407423"),
+      Period("2086", "Q3").get,
```

Comment:
You can use Period(2086, Q3) so that it doesnt return a try and no need for .get

Reply:
Changed to recommendation.